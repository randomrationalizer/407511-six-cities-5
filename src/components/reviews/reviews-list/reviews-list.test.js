import React from "react";
import renderer from "react-test-renderer";
import ReviewsList from "./reviews-list";
import mockReviews from "../../../mocks/test-data/reviews";


const reviews = mockReviews;


test(`Should Reviews List component renders correctly`, () => {
  const tree = renderer
    .create(
        <ReviewsList
          reviews={reviews}
        />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});

