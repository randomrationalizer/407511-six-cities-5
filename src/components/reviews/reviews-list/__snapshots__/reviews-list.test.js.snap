// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Reviews List component renders correctly 1`] = `
<ul
  className="reviews__list"
>
  <li
    className="reviews__item"
  >
    <div
      className="reviews__user user"
    >
      <div
        className="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          className="reviews__avatar user__avatar"
          height="54"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/avatar/9.jpg"
          width="54"
        />
      </div>
      <span
        className="reviews__user-name"
      >
        Sophie
      </span>
    </div>
    <div
      className="reviews__info"
    >
      <div
        className="reviews__rating rating"
      >
        <div
          className="reviews__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "60%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        className="reviews__text"
      >
        We loved it so much, the house, the veiw, the location just great.. Highly reccomend :)
      </p>
      <time
        className="reviews__time"
        dateTime="2020-11-02"
      >
        November 2020
      </time>
    </div>
  </li>
  <li
    className="reviews__item"
  >
    <div
      className="reviews__user user"
    >
      <div
        className="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          className="reviews__avatar user__avatar"
          height="54"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/avatar/7.jpg"
          width="54"
        />
      </div>
      <span
        className="reviews__user-name"
      >
        Mollie
      </span>
    </div>
    <div
      className="reviews__info"
    >
      <div
        className="reviews__rating rating"
      >
        <div
          className="reviews__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "40%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        className="reviews__text"
      >
        What an amazing view! The house is stunning and in an amazing location. The large glass wall had an amazing view of the river!
      </p>
      <time
        className="reviews__time"
        dateTime="2020-11-02"
      >
        November 2020
      </time>
    </div>
  </li>
</ul>
`;
